src/main/resources/public/jsx/Form.jsx

```diff
@@ -101,16 +101,16 @@ const Checkbox = ({ handleCheckboxChange, label, value, checked }) =>
     </div>
     
 
-const SubmitButton = ({ label, onSubmit, disabled }) =>
+const SubmitButton = ({ label, onClick, disabled }) =>
```

Comment:
I've not seen this onClick used in this PR ?

Reply:
removed property